zh/deploy-on-aws-eks.md

```diff
@@ -269,6 +269,14 @@ region = us-west-21
     kubectl --kubeconfig credentials/kubeconfig_${eks_name} create -f db-monitor.yaml -n ${namespace}
     ```
 
+4. 查看 TiDB 集群启动状态：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_${eks_name} get pods -n ${namespace}
+    ```
+
```

Comment:
Suggest adding an expected output here, so users can confirm whether the deployment is successful or not.

Reply:
addressed in https://github.com/pingcap/docs-tidb-operator/pull/523/commits/2285f1a658d5e2f748f3ef53eb76602752019955